### PR TITLE
Fixing the display of sessionsSubmitted in dashboard

### LINKED ISSUE #1882: Event Dashboard: Speaker's data and design incorrect or missing
Please see screenshot below. There are a number of issues, e.g. Submitted number is missing.

Please ensure each data is in one line and type and numbers have the same distance. Also ensure all numbers are correct and showing up.

This issue only appears if the user has an event with data setup at https://eventyay.com/events/live

screenshot from 2019-01-18 00-24-43

### PR DESCRIPTION
**Pull request by uds5501**
- Created a local component ``cell-sessions-dashboard``
- Renamed the ``event-statistics-general`` attribute ``sessionsSubmited`` to ``sessionsSubmitted``
- Modified ``event`` controller to route to this new component
- Created a component integration test along the lines of existing ``cell-sessions`` test

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
In the dashboard, Submitted Sessions was not showing up, this PR resolves that bug 

#### Changes proposed in this pull request:

- Creation of a new local component ``cell-sessions-dashboard``
- Renaming the ``event-statistics-general`` attribute ``sessionsSubmited`` to ``sessionsSubmitted``
- Modifying ``event`` controller to route to this new component

UI Fix : 
image

Fixes #1882

### CODE REVIEW

**Comment by uds5501** [comment]
@mariobehling Please review

File: app/components/ui-table/cell/cell-sessions-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+});
```
Comment (ritikamotwani) [inline app/components/ui-table/cell/cell-sessions-dashboard.js]: @uds5501 delete this file. It's just an empty file, not required.

**Comment by uds5501** [comment]
@pradeepgangwar @ritikamotwani I did as requested. it should do now

**Comment by CosmicCoder96** [comment]
@uds5501 Please ensure you squash your commits in future when sending a PR.

**Comment by uds5501** [comment]
@CosmicCoder96 I will surely do that sir

**Pull request merged**